src/pages/heeyeolKim/Main/components/Story.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+
+import './Story.scss'
+
+class Story extends Component {
+
+    render() {
+        const { storyImage, storyName } = this.props;
+        return (
+            <div>
+                <li><img src={storyImage} alt="사진을 찾을 수 없습니다."/></li>
+                {storyName}
+            </div>
```

Comment:
```suggestion
            <>
                <li>
                    <img src={storyImage} alt="사진을 찾을 수 없습니다."/>
                </li>
                <span>{storyName}</span>
            </>
```
- 좀 더 명확한 요소 구분을 위해 span 태그를 사용해봤습니다. (개발자마다 스타일이 다를 수 있습니다!)
- React Fragment 를 활용해보세요!